src/oscar/core/thumbnails.py

```diff
@@ -1,7 +1,11 @@
+import logging
+
 from django.apps import apps
 from django.conf import settings
 from django.utils.module_loading import import_string
 
+logger = logging.getLogger(__name__)
```

Comment:
We generally provide explicit namespaces for all our Oscar loggers - think we should do the same here and use `oscar.thumbnails`.

Reply:
Done.